docs/content/en/docs-dev/user-guide/managing-application/configuration-drift-detection.md

```diff
@@ -52,6 +52,8 @@ This status means the application is deploying and the configuration drift detec
 
 This feature is automatically enabled for all applications.
 
+If you want to deploy automatically when `OUT_OF_SYNC` occures, configure each application's [OnOutOfSync](../configuration-reference/#onoutofsync).
```

Comment:
```suggestion
If you want to trigger deployment automatically when `OUT_OF_SYNC` occurs, configure each application's [OnOutOfSync](../configuration-reference/#onoutofsync).
```
typo and re-write 👀

Reply:
Same for the others

Reply:
On the second thought, I think this note could lead to confusing situation 🤔 We have place explain how to trigger pipecd deployment here: https://pipecd.dev/docs-v0.48.x/user-guide/managing-application/triggering-a-deployment/#trigger-configuration
Merging this note here could lead to confusion between drift detection feature and trigger feature of pipecd. WDYT? @t-kikuc @Warashi

Reply:
This document is not a document like a dictionary, so we can write the same thing on several pages.
However, we can avoid confusion by the guide having centralized, with only one page at one thing.
I propose that this notation have a link to the page @khanhtc1202 mentioned, not the configuration reference page.

Reply:
>I propose that this notation have a link to the page @khanhtc1202 mentioned, not the configuration reference page.

I agree

Reply:
I fixed: [a25e438](https://github.com/pipe-cd/pipecd/pull/5120/commits/a25e43839290cef322b4d39cac548c5435850ba9)

```markdown
### How to enable

This feature is automatically enabled for all applications.

If you want to trigger deployment automatically when `OUT_OF_SYNC` occurs, see [Trigger configuration](./triggering-a-deployment/#trigger-configuration).
```